File: Beecrowd/3093 Truco da Galera 1.0/main.c
#include <stdio.h>
#include <string.h>

int main(){
    int n, cont;
    char str[1001];

    scanf("%d", &n);

    while(n--){
        scanf("%s", str);

        if((strchr(str, 'Q') && strchr(str, 'J') && strchr(str, 'K') && strchr(str, 'A')) != NULL) // strchr() ve se tem o carater passado no parametro na string
            printf("Aaah muleke\n");
        else
            printf("Ooo raca viu\n");
    }

    return 0;
}

File: Beecrowd/3093 Truco da Galera 1.0/Main.java
import java.io.IOException;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) throws IOException {
        Scanner sc = new Scanner(System.in);

        int n;
        String str;

        n = sc.nextInt();

        for (int i = 0; i < n; i++) {
            str = sc.next();

            if (str.contains("Q") && str.contains("J") && str.contains("K") && str.contains("A")) {
                System.out.println("Aaah muleke");
            } else
                System.out.println("Ooo raca viu");
        }

        sc.close();
    }

}